src/component/menu/routes.js

```diff
@@ -242,6 +243,16 @@ export const routes = [
         layout: 'main',
         menu: {},
     },
+    {
```

Comment:
I guess we plan to replace/rename these routes to just features once we're satisfied we've landed the correct view?

Reply:
Instead of `features2` that is